docs/user/filter-and-process/custom-pipeline-processing-ottl.md

```diff
@@ -0,0 +1,161 @@
+- **conditions** (optional): A list of OTTL conditions. If you provide conditions, the statements only run if at least one of the conditions is true
+
+### Example: General Resource Attribute Enrichment
+
+This example adds a `deployment.environment` attribute with the value `production` to all metrics in the pipeline. Since there are no conditions, the rule applies to all data.
+
+```yaml
+# In your MetricPipeline spec
+spec:
+  input:
+    prometheus:
+      enabled: true
+  output:
+    otlp:
+      endpoint: http://metrics.example.com:4317
+  transform:
+    - statements:
+        - 'set(resource.attributes["deployment.environment"], "production")'
+```
+
+### Example: Conditional Resource Attribute Enrichment
+
+This example sets the status code of a trace span to `1` (Error) if its pod name matches `my-pod-name.*` and its `http.path` attribute is `/health`.
+
+```yaml
+# In your TracePipeline spec
+spec:
+  output:
+    otlp:
+      endpoint: http://traces.example.com:4317
+  transform:
+    - conditions:
+        - 'IsMatch(resource.attributes["k8s.pod.name"], "my-pod-name.*")'
+      statements:
+        - 'set(span.status.code, 1) where span.attributes["http.path"] == "/health"'
+```
+
+## Filtering Telemetry Data
+
+Use filters to drop unwanted telemetry data from a pipeline. Filtering helps you reduce noise, lower storage and processing costs, and focus on the data that matters most.
+
+You configure filters in the `filter` section of a pipeline's spec. The filter processor drops any data point, span, or log record that matches one of your conditions.
+
+Each filter block consists of a list of conditions. If multiple conditions are provided within a filter block, they are combined with a logical `OR` (any matching condition drops the data).
+
+### Example: Drop Debug Logs
+
+This example drops any log record that has a severity level below warning (e.g., debug and info logs).
+
+```yaml
+# In your LogPipeline spec
+spec:
+  input:
+    application:
+      enabled: true
+  output:
+    otlp:
+      endpoint: http://logs.example.com:4317
+  filter:
+    - conditions:
+        - 'severity_number < SEVERITY_NUMBER_WARN'
+```
+
+### Filter envoy metrics by outlier_detection only
+
+This example keeps only Envoy outlier detection metrics, which are essential for monitoring circuit breaker behavior and host ejections in your service mesh. All other metrics are dropped to reduce noise.
+
+```yaml
+# In your MetricPipeline spec
+spec:
+  input:
+    istio:
+      enabled: true
+      envoyMetrics:
+        enabled: true
+  output:
+    otlp:
+      endpoint: http://metrics.example.com:4317
+  filter:
+    - conditions:
+        - 'IsMatch(metric.name, ".*outlier_detection.*") == false'
+```
+
+## Limitations
+
+Review the following limitations when constructing your processing rules.
+
+### Explicit Context Path Required
+
+You must specify the full context path for attributes. The system uses the lowest-level context by default:
```

Comment:
Again, what means by default. We say you must specify the context, so there is no default